relacy/relacy_std.hpp

```diff
@@ -19,64 +19,68 @@
 

```

Comment:
examples/peterson/perterson.cpp does not build with this patch, because it uses rl::memory_order_relaxed which gets redefined into rl::rlimpl::memory_order_relaxed, add:
```
namespace rl
{
    namespace rlimp
    {
        using namespace rl;
    }
}
```
to fix it.